libeos-payg/efi.c

```diff
@@ -474,6 +484,65 @@ eospayg_efi_secureboot_active (void)
   return !!content[0];
 }
 
+/* eospayg_efi_securebootoption_disabled:
+ *
+ * Check if the SecureBootOption EFI variable exists and
+ * is disabled.
+ *
+ * The oddly inverted logic is due to the fact that most
+ * systems don't have this variable at all - if it doesn't
+ * exist, we can infer nothing about the state of Secure Boot,
+ * if it does it tells us if Secure Boot is on or off.
```

Comment:
We should probably try to be extra clear here and go with something like:
```suggestion
 * if it does it tells us if the Secure Boot option in the
 * BIOS is enabled or disabled.
```